scripts/core/kernel.yml

```diff
@@ -52,7 +49,24 @@ params:
       desc: "[in] size of argument type"
     - type: "const void*"
       name: pArgValue
-      desc: "[in][optional] argument value represented as matching arg type. If null then argument value is considered null."
+      desc: "[in] argument value represented as matching arg type."
+--- #--------------------------------------------------------------------------
+type: function
```

Comment:
Do we want to be able to set a local memory argument to NULL through this API? And if so how would that work?

Reply:
Is this possible in OpenCL?

Reply:
I don't think so. In OpenCL you'd call `clSetKernelArg` with `NULL` as the argument and the required size of the local memory buffer in bytes in the size argument. So you could set the buffer size to zero (you could also do that here), but I don't think there is any way to actually set it to `NULL.`

To set a  global buffer to `NULL` in OpenCL you set the argument to `NULL` and the size to `sizeof(cl_mem)`.

Reply:
I've left this as it is since it is basically equivalent to what OpenCL has.